lib/main.dart

```diff
@@ -0,0 +1,101 @@
+import 'dart:math';
+
+import 'package:dart_collections_journey/names.dart';
+import 'package:word_generator/word_generator.dart';
+
+void main(List<String> args) {
+  runTask1();
+  runTask2();
+  runTask3();
+}
+
+void runTask1() {
+  printSeperator("Task 1");
+
+  List<int> numbers = List.generate(100, (el) => Random().nextInt(101));
+  print(numbers);
+  print("\n65-й елемент: ${numbers[64]}");
+
+  // Вставте число 1000000000 на 50-ту позицію списку.
+  numbers.insert(51, 1000000000);
+
+  // Видаліть зі списку елементи зі значеннями: 24, 45, 66, 88.
+
+  List<int> numbersForRemove =
```

Comment:
You can mark it final
```suggestion
  final List<int> numbersForRemove =
```